streamalert/athena_partition_refresh/main.py

```diff
@@ -205,17 +203,16 @@ def run(self, event):
                 # Account for special characters in the S3 object key
                 # Example: Usage of '=' in the key name
                 object_key = urllib.parse.unquote_plus(s3_rec['s3']['object']['key']).encode()
+                if object_key.endswith(b'_$folder$'):
```

Comment:
why the `b` prefix here? shouldn't this be a string not bytes?

Reply:
Encode returns a byte array and the argument to endswith has to match that type.